anybox/recipe/odoo/runtime/session.py

```diff
@@ -7,29 +7,20 @@
 
 
 try:
-    import openerp as odoo
+    import odoo
 except ImportError:
     try:
-        import odoo
+        import openerp as odoo
     except ImportError:
         warnings.warn("This must be imported with a buildout odoo recipe "
                       "driven sys.path", RuntimeWarning)
-    else:
-        try:
-            from odoo.cli import server as startup
-        except ImportError:
-            from .backports.cli import server as startup
-        from odoo.tools import config
-        from odoo import SUPERUSER_ID
-        from odoo.tools.parse_version import parse_version
-else:
-    try:
-        from openerp.cli import server as startup
-    except ImportError:
-        from .backports.cli import server as startup
-    from openerp.tools import config
-    from openerp import SUPERUSER_ID
-    from openerp.tools.parse_version import parse_version
+try:
+    from odoo.cli import server as startup
+except ImportError:
+    from .backports.cli import server as startup
+from odoo.tools import config
```

Comment:
Python appear to not be able to load python module from aliases:
```python
>>> from xml.dom import domreg
>>> domreg
<module 'xml.dom.domreg' from '/usr/lib/python2.7/xml/dom/domreg.pyc'>
>>> import xml as tmp
>>> from tmp.dom import domreg
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: No module named tmp.dom
>>> 
```

I've test that which works in our case

@PieterPaulussen what do you think about?

```python
try:
    import openerp as odoo
except ImportError:
    try:
        import odoo
    except ImportError:
        warnings.warn("This must be imported with a buildout odoo recipe "
                      "driven sys.path", RuntimeWarning)
try:
    startup = odoo.cli.server
except ImportError:
    from .backports.cli import server as startup
config = odoo.tools.config
SUPERUSER_ID = odoo.SUPERUSER_ID
parse_version = odoo.tools.parse_version
```